compiler/test/dotty/tools/scripting/ClasspathTests.scala

```diff
@@ -56,8 +56,7 @@ class ClasspathTests:
     printf("bash is [%s]\n", bashExe)
 
     if packBinScalaExists then
-      val echoTest = "" // "SCALAC_ECHO_TEST=1"
-      val bashCmdline = s"SCALA_OPTS= $echoTest ${scalaCopy.norm} -classpath '$wildcardEntry' $relpath"
+      val bashCmdline = s"SCALA_OPTS= ${scalaCopy.norm} -classpath '$wildcardEntry' $relpath"
```

Comment:
Why scala script path is part of `SCALA_OPTS`?

Reply:
The purpose of prefixing the bash command line with `SCALA_OPTS=` is to unset SCALA_OPTS, to insure that it doesn't influence the command line seen by `dist/bin/scala`. (there's a space following the equal sign, effectively unsetting SCALA_OPTS)

Reply:
Oh yeah, I was curious about the blank space too, but now it makes sense